sdk/core/azure-core/inc/azure/core/uuid.hpp

```diff
@@ -33,9 +33,9 @@ namespace Azure { namespace Core {
   public:
     /**
      * @brief Gets Uuid as a string.
-     * @details A string is in canonical format (4-2-2-2-6 lowercase hex and dashes only).
+     * @details A string is in canonical format (8-4-4-4-12 lowercase hex and dashes only).
      */
-    std::string ToString();
+    std::string ToString() const;
```

Comment:
Nice, I've just recently ran into this in the codegen (https://github.com/Azure/autorest.cpp/pull/378).

Can you add it to bug fixes readme section.
(to clarify, this is not a breaking change)